src/main/java/model/LottoNum.java

```diff
@@ -0,0 +1,50 @@
+package model;
+
+import valueObject.ErrorMessage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class LottoNum {
+    private static final int LOTTO_PRICE = 1000;
+    private static final int LOTTO_SIZE = 6;
+
+    private ErrorMessage errorMessage = new ErrorMessage();
+    // 금액을 입력받고 몇 장을 샀는지 확인하는 메서드
+    public int buyLotto(int money) {
+        if (money%1000 != 0) {
+            throw new IllegalArgumentException(errorMessage.ERROR_PRICE);
+        }
+        return money/LOTTO_PRICE;
+    }
+
+    // 랜덤으로 숫자를 뽑아 리스트에 넣어주는 메서드
+    public List<Integer> lottoNumbers() {
```

Comment:
주석으로 해당 메소드가 어떤 기능을 가졌는지, 설명해주시고 계신데 코드만 보고 어떤 기능인지 한눈에 파악할 수 있어야 좋은 코드라고 말할 수 있습니다.
이 메소드의 문제점은 메소드의 메소드명이 코드 기능을 전혀 나타내고 있지 않습니다.
보통 메소드 명은 동사 + 명사  혹은 동사로 지어집니다. 적절하게 수정 부탁드립니다.

```suggestion
    public List<Integer> create() {
```
LottoNum 클래스 내부에 있기에 명사를 생략했습니다. 당연히 LottoNum을 만들 걸 알기에!